### PR TITLE
Updated deep learning notebooks - 1

### PR DESCRIPTION
**Pull request by shivanip32**
\<insert pull request description here\>

-----

# Checklist

Please go through each entry in the below checklist and mark an 'X' if that condition has been met. Every entry should be marked with an 'X' to be get the Pull Request approved.

- [ ] All `import`s are in the first cell? 
    - [ ] First block of imports are standard libraries
    - [ ] Second block are 3rd party libraries
    - [ ] Third block are all `arcgis` imports? Note that in some cases, for samples, it is a good idea to keep the imports next to where they are used, particularly for uncommonly used features that we want to highlight.
- [ ] All `GIS` object instantiations are one of the following?
    - `gis = GIS()`
    - `gis = GIS('home')` or `gis = GIS('pro')`
    - `gis = GIS(profile="your_online_portal")`
    - `gis = GIS(profile="your_enterprise_portal")`
- [ ] If this notebook requires setup or teardown, did you add the appropriate code to `./misc/setup.py` and/or `./misc/teardown.py`?
- [ ] If this notebook references any portal items that need to be staged on AGOL/Python API playground, did you coordinate with a Python API team member to stage the item the correct way with the `api_data_owner` user?
- [ ] If the notebook requires working with local data (such as CSV, FGDB, SHP, Raster files), upload the files as items to the [Geosaurus Online Org](geosaurus.maps.arcgis.com/) using `api_data_owner` account and change the notebook to first download and unpack the files.
- [ ] Code simplified & split out across multiple cells, useful comments?
- [ ] Consistent voice/tense/narrative style? Thoroughly checked for typos?
- [ ] All images used like `` instead of ``? All map widgets contain a static image preview? (Call `mapview_inst.take_screenshot()` to do so)
- [ ] All file paths are constructed in an OS-agnostic fashion with `os.path.join()`? (Instead of `r"\foo\bar"`, `os.path.join(os.path.sep, "foo", "bar")`, etc.)
- [ ] Is your code formatted using [Jupyter Black](https://www.freecodecamp.org/news/auto-format-your-python-code-with-black/)? You can use Jupyter Black to format your code in the  notebook.
- [ ] **IF YOU WANT THIS SAMPLE TO BE DISPLAYED ON THE DEVELOPERS.ARCGIS.COM WEBSITE**, ping @ mohi9282 so he can add it to the list for the next deploy

### CODE REVIEW

File: samples/04_gis_analysts_data_scientists/address-standardization-and-correction-with-sequencetosequence.ipynb [hunk 1]
```diff
@@ -1 +1,1031 @@
... city, <span style=\"background-color: LightGreen\">ia</span>, 50401, us</span>\n", "\n"]}, {"cell_type": "markdown", "metadata": {}, "source": ["# References"]}, {"cell_type": "markdown", "metadata": {}, "source": ["[1] [Learning Rate](https://en.wikipedia.org/wiki/Learning_rate)\n", "\n", "[2] [Accuracy](https://en.wikipedia.org/wiki/Accuracy_and_precision)\n", "\n", "[3] [Bleu score](https://en.wikipedia.org/wiki/BLEU)"]}], "metadata": {"esriNotebookRuntime": {"notebookRuntimeName": "ArcGIS Notebook Python 3 Advanced with GPU support", "notebookRuntimeVersion": "5.0"}, "kernelspec": {"display_name": "Python 3", "language": "python", "name": "python3"}, "language_info": {"codemirror_mode": {"name": "ipython", "version": 3}, "file_extension": ".py", "mimetype": "text/x-python", "name": "python", "nbconvert_exporter": "python", "pygments_lexer": "ipython3", "version": "3.7.9"}, "toc": {"base_numbering": 1, "nav_menu": {}, "number_sections": true, "sideBar": true, "skip_h1_title": false, "title_cell": "Table of Contents", "title_sidebar": "Contents", "toc_cell": false, "toc_position": {}, "toc_section_display": true, "toc_window_display": false}}, "nbformat": 4, "nbformat_minor": 4}
```
[the remark this answers is not in the document]
Comment (shivanip32) [reply samples/04_gis_analysts_data_scientists/address-standardization-and-correction-with-sequencetosequence.ipynb:1]: <p>which portal are you using to run this notebook?</p>

[the remark this answers is not in the document]
Comment (shivanip32) [reply samples/04_gis_analysts_data_scientists/address-standardization-and-correction-with-sequencetosequence.ipynb:1]: <p>marked it as comment, if can be used if the user needs to install the library.</p>

[the remark this answers is not in the document]
Comment (shivanip32) [reply samples/04_gis_analysts_data_scientists/address-standardization-and-correction-with-sequencetosequence.ipynb:1]: <p>Yes</p><p> </p>

File: samples/04_gis_analysts_data_scientists/detecting_swimming_pools_using_automated_machine_learning.ipynb [hunk 2]
```diff
@@ -62,7 +62,7 @@
   },
```
[the remark this answers is not in the document]
Comment (shivanip32) [reply samples/04_gis_analysts_data_scientists/detecting_swimming_pools_using_automated_machine_learning.ipynb:62]: <p>I have added one line which mentions to provide the full path of the .emd by referring the table.</p>

File: samples/04_gis_analysts_data_scientists/detecting_mussel_farms_using_deep_learning.ipynb [hunk 3]
```diff
@@ -2,17 +2,17 @@
  "cells": [
```
[the remark this answers is not in the document]
Comment (shivanip32) [reply samples/04_gis_analysts_data_scientists/detecting_mussel_farms_using_deep_learning.ipynb:2]: <p>works fine</p>

**Comment by priyankatuteja** [comment]
New PR will be raised including map widget changes

**Pull request closed without merging**